modules/apps/configuration-admin/configuration-admin-test/src/testIntegration/java/com/liferay/configuration/admin/internal/search/test/BaseConfigurationModelTest.java

```diff
@@ -0,0 +1,377 @@
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+package com.liferay.configuration.admin.internal.search.test;
+
+import aQute.bnd.annotation.metatype.Meta;
+
+import com.liferay.osgi.util.service.OSGiServiceUtil;
+import com.liferay.petra.string.StringPool;
+import com.liferay.portal.configuration.metatype.definitions.ExtendedAttributeDefinition;
+import com.liferay.portal.configuration.metatype.definitions.ExtendedObjectClassDefinition;
+import com.liferay.portal.kernel.model.CompanyConstants;
+import com.liferay.portal.kernel.search.Document;
+import com.liferay.portal.kernel.search.Hits;
+import com.liferay.portal.kernel.search.IndexWriterHelper;
+import com.liferay.portal.kernel.search.Indexer;
+import com.liferay.portal.kernel.search.SearchContext;
+import com.liferay.portal.kernel.search.SearchException;
+import com.liferay.portal.kernel.util.LocaleThreadLocal;
+import com.liferay.portal.kernel.util.LocaleUtil;
+import com.liferay.portal.test.rule.Inject;
+
+import java.io.InputStream;
+
+import java.lang.reflect.Constructor;
+import java.lang.reflect.InvocationTargetException;
+import java.lang.reflect.Method;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Locale;
+import java.util.Map;
+import java.util.Set;
+
+import org.junit.After;
+import org.junit.Assert;
+import org.junit.Before;
+
+import org.osgi.framework.Bundle;
+import org.osgi.framework.BundleContext;
+import org.osgi.framework.FrameworkUtil;
+import org.osgi.service.cm.Configuration;
+import org.osgi.service.cm.ConfigurationAdmin;
+
+/**
+ * @author Vagner B.C
+ */
+public class BaseConfigurationModelTest {
+
+	@Before
+	public void setUp() throws Exception {
+		reflectionConfigurationModel();
+
+		defaultLocale = LocaleThreadLocal.getDefaultLocale();
+	}
+
+	protected void reflectionConfigurationModel()
+		throws ClassNotFoundException, NoSuchMethodException {
+
+		bundle = FrameworkUtil.getBundle(ConfigurationModelIndexerTest.class);
+
+		bundleContext = bundle.getBundleContext();
+
+		Bundle configAdminWebBundle = getBundle(
+			"com.liferay.configuration.admin.web");
+
+		Class<?> configurationModelClass = configAdminWebBundle.loadClass(
+			"com.liferay.configuration.admin.web.internal.model." +
+				"ConfigurationModel");
+
+		configurationModelConstructor = configurationModelClass.getConstructor(
+			ExtendedObjectClassDefinition.class, Configuration.class,
+			String.class, String.class, boolean.class);
+	}
+
+	@After
+	public void tearDown() throws SearchException {
```

Comment:
I've noted the methods are not in visibility-then-alphabetical order. And source formatter did not fixed it either :thinking: If source format does not fix it to you, could you reorder it?